api/resources/translations/messages-en.properties

```diff
@@ -1307,6 +1309,9 @@ user.password.current = Current password
 user.phone.help = Must be a valid phone number without dashes or punctuation.
 user.place.help = All users must be associated with a place in the system to be correctly located in the hierarchy and see appropriate data when they log in.
 user.username.help = This is what you will use to log in to the app.
+user.sso.username = SSO Email Address
```

Comment:
Note that we are intentionally calling this field in the admin UI "SSO Email Address" even though it populates the `oidc_username` field on the backend.  This is because for the current MVP implementation, this value _must_ map to the `email` claim value provided by the OIDC provider.  It is possible we could support different mapping in the future, but for now it seems most clear to just call this an "Email Address".